### PR TITLE
drop PRM api calls

### PR DESCRIPTION
**Pull request by AlexKovalevych**
https://github.com/edenlabllc/ehealth.api/issues/735
https://github.com/edenlabllc/ehealth.api/issues/856
https://github.com/edenlabllc/ehealth.api/issues/882
https://github.com/edenlabllc/ehealth.api/issues/883

### CODE REVIEW

File: lib/ehealth/declaration_request/api.ex [hunk 1]
```diff
@@ -80,14 +79,15 @@ defmodule EHealth.DeclarationRequest.API do
...
+         %LegalEntity{} = legal_entity <- LegalEntities.get_legal_entity_by_id_with(
+                                            client_id,
+                                            :medical_service_provider
+                                          ),
+         %Division{} = division <- Divisions.get_division_by_id(attrs["division_id"])
+    do
       updates = [status: "CANCELLED", updated_at: DateTime.utc_now(), updated_by: user_id]
       global_parameters = GlobalParameters.get_values()
 
-      employee = PRMRepo.preload(employee, [:party, :doctor])
```
Comment (gmile) [inline lib/ehealth/declaration_request/api.ex]: Where did this go?

I think it's premature to have doctor preloaded in `get_employee_by_id` call. Reason: there will definitely be more types of employees (in fact there are now, e.g. I think some form of OWNER kind of employee must exist).

By incorporating preloading `doctor` in a function with generic name, such as `get_employee_by_id`, we're making it harder for future changes, when we'd need to load employee and preload some other specific association, something other than doctor.

I'd suggest we rename `get_employee_by_id` call to something like `get_doctor_by_employee_id`, and leave `get_by_employee_id` for more generic use.

File: lib/ehealth/legal_entity/api.ex [hunk 2]
```diff
@@ -17,12 +17,16 @@ defmodule EHealth.LegalEntity.API do
...
   alias EHealth.PRM.LegalEntities
   alias EHealth.PRM.LegalEntities.Schema, as: LegalEntity
+  alias Ecto.Schema.Metadata
 
   require Logger
 
   @employee_request_status "NEW"
   @employee_request_type "OWNER"
 
+  @status_closed "CLOSED"
```
Comment (gmile) [inline lib/ehealth/legal_entity/api.ex]: :+1:

File: lib/ehealth/declaration_request/api.ex (hunk 1 above)
Comment (AlexKovalevych) [inline lib/ehealth/declaration_request/api.ex]: https://github.com/edenlabllc/ehealth.api/pull/849/files/90982469fa8d51574918531127f2e1ff69cb820a#diff-4b57ced23d8e589b2436ffc2985061baR55

Comment (gmile) [inline lib/ehealth/declaration_request/api.ex]: @AlexKovalevych yeah, I figured where this code moved. I'm curious why are we making preloading `doctor` entity for all employees. Even for those who are not doctors. We might not have such non-doctor employees now, but will have them in future.

Comment (Troush) [inline lib/ehealth/declaration_request/api.ex]: @gmile We will merge `employee_doctor` to `employee` in near future. So we can just leave it like it was before.

File: lib/tasks/tasks.ex [hunk 3]
```diff
@@ -8,24 +8,25 @@ defmodule :ehealth_tasks do
...
   def migrate! do
+    prm_migrations_dir = Path.join(["priv", "prm_repo", "migrations"])
     migrations_dir = Path.join(["priv", "repo", "migrations"])
 
+    load_app()
+
+    prm_repo = EHealth.PRMRepo
+    prm_repo.start_link()
+
+    Ecto.Migrator.run(prm_repo, prm_migrations_dir, :up, all: true)
```
Comment (PavelVesnin) [inline lib/tasks/tasks.ex]: 👍

**Comment by Troush** [comment]
🎉

**Pull request merged**